docs/gallery/howto/autogen/aggregate.py

```diff
@@ -0,0 +1,309 @@
+"""
+==================================
+Aggregate data from multiple tasks
+==================================
+"""
+# %%
+# Introduction
+# ============
+# In this tutorial, you will learn how to aggregate data from the outputs of
+# multiple tasks by linking tasks or by using context. Then we discuss at the
+# end how to deal with nested datatypes.
+#
+# Load the AiiDA profile.
+#
+
+
+from aiida import load_profile
+
+load_profile()
+
+
+# %%
+# Using multi-linking for dynamic inputs
+# ======================================
+# In the following example, we create multiple tasks that return a random
+# integer, and then aggregate all the results and calculate the sum by linking
+# the inputs of the tasks to the input of one final task
+
+#
+
+
+from aiida_workgraph import task, WorkGraph
+from aiida.orm import Int
+
+
+@task.calcfunction()
+def generator(seed: Int):
+    import random
+
+    random.seed(seed.value)
+    return Int(random.randint(0, 1))  # one can also use
+
+
+@task.calcfunction()
+def aggregate(**collected_values):  # We use a keyword argument to obtain a dictionary
+    for key, value in collected_values.items():
+        print("key:", key, "value:", value)
+    return {"sum": sum(collected_values.values())}
+
+
+wg = WorkGraph("aggregate_by_multilinking")
+
+aggregate_task = wg.add_task(aggregate, name="aggregate_task")
+
+# we have to increase the link limit because by default workgraph only supports one link per input socket
+# this is still an experimental feature that is why
```

Comment:
I added the reason at the beginning.
```suggestion
```

Reply:
```python
def add(x, y):
    return x + y

# Adding a limit on the number of allowed inputs can prevent users from passing multiple values
# into a single input by mistake, such as in the example below:
add(x=1, x=2, y=3)
```
This type of mistake is easy to avoid when writing code, but it can be difficult to spot when using links, especially in large and complex workflows.